docker/caffe2/jenkins/common/install_rocm.sh

```diff
@@ -41,20 +45,60 @@ install_ubuntu() {
     sed -i 's/find_dependency(hip)/find_dependency(HIP)/g' /opt/rocm/rocblas/lib/cmake/rocblas/rocblas-config.cmake
 }
 
+install_devtoolset() {
```

Comment:
I meant putting it in a separate shell script (so other non-rocm docker images can use it to control devtoolset as well)

Reply:
OK.